packages/nimble-components/src/table/components/cell/index.ts

```diff
@@ -32,6 +32,12 @@ export class TableCell<
     @observable
     public recordId?: string;
 
+    @attr({ attribute: 'is-parent-row', mode: 'boolean' })
+    public isParentRow = false;
```

Comment:
If we go that direction (I personally agree) then `isInParentRow` might be even better since it keeps the "is..." naming pattern for boolean properties.